code/modules/reagents/chemistry/reagents/water.dm

```diff
@@ -339,7 +340,7 @@
 	drink_icon = "glass_clear"
 	drink_name = "Glass of Water"
 	drink_desc = "The father of all refreshments."
-	taste_message = null
+	taste_message = "holiness"
```

Comment:
You shouldn't be able to distinguish holy water from normal water by merely tasting it

Reply:
That can certainly be changed, though I don't really think it matters personally. If it matters to you, the fact that you're tasting it means it's already waaay too late. It should taste like hellfire to creatures it harms anyway.